plugin/deoplete-ternjs.vim

```diff
@@ -4,7 +4,9 @@ endif
 
 let g:loaded_deoplete_ternjs = 1
 
-let g:deoplete#sources#ternjs#tern_bin = get(g:, 'deoplete#sources#ternjs#tern_bin', 'tern') 
+call ternjs#Enable()
```

Comment:
I think the function should be called in `on_init` in source.

Reply:
Maybe. Anyway I am working on different solution.